content/css/jenkins.css

```diff
@@ -1761,3 +1761,10 @@ i.icon-report::before {
   vertical-align: top;
   margin-right: 5px;
 }
+
+.version {
+  background: hsl(0deg, 0%, 93%);
+  font-size: 13px;
+  font-weight: 600;
+  padding: 1.75ch 2ch;
+}
```

Comment:
This class is used on https://updates.jenkins.io/download/ pages, and already got a style in https://github.com/jenkins-infra/jenkins.io/blob/master/content/templates/downloads.html.haml, and this change makes it look really weird.

Why is this a global class rather than being more limited in scope?

Reply:
Ah shoot, I did have a look but my scope was limited to `.css` so I didn't spot this one. I've opened a small PR here to fix it https://github.com/jenkins-infra/jenkins.io/pull/4953 - thanks for finding :)